### PR TITLE
remove withdrawn req

### PR DESCRIPTION
**Pull request by RoriCremer**
Allow CH to create custom cohorts with a subset of samples

Tested with sample_id=1 (sample name = ERS4367795)

Imported:
Screen Shot 2022-05-13 at 4 13 15 PM

In the alt-allele
Screen Shot 2022-05-17 at 6 52 30 PM

In the filter:
Screen Shot 2022-05-17 at 6 51 43 PM

In the prepare:
Screen Shot 2022-05-17 at 2 47 32 PM

In the extract:
Screen Shot 2022-05-17 at 6 48 17 PM

### CODE REVIEW

**Comment by kcibul** [comment]
As part of this, do we want to also remove this restriction in the other places we put it in?   A quick grep turns up

GvsExtractCallset.wdl
GvsCreateFilterSet.wdl
populate_alt_allele_table.py
GvsImportGenomes.wdl

**Comment by RoriCremer** [comment]
Bec and I were just having this discussion---do we want to just unblock CH asap or do we want to remove this everywhere---but given that we haven't unblocked him yet and seem to be blocked on the date / timestamp---yes, let's remove them all!

File: scripts/variantstore/wdl/GvsExtractCallset.wdl
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsExtractCallset.wdl:395]: do we want to retain this?

Comment (gbggrant) [reply scripts/variantstore/wdl/GvsExtractCallset.wdl:395]: I think you should. It's entirely dependent on the database state.

**Review by gbggrant — COMMENTED** [review]
I think the volatile = true *should* be retained.

File: scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py
Comment (kcibul) [inline scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py:92]: separate discussion, but I wonder if this encoding of "control" samples is another case where we put biz logic into GVS?

**Review by kcibul — APPROVED** [review]

File: scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py
Comment (RoriCremer) [reply scripts/variantstore/wdl/extract/create_ranges_cohort_extract_data_table.py:92]: maybe---but the biz logic is OURS for sensitivity and precision

File: scripts/variantstore/wdl/GvsExtractCallset.wdl
Comment (RoriCremer) [reply scripts/variantstore/wdl/GvsExtractCallset.wdl]: ok---no longer removed as part of the pr!

**Comment by RoriCremer** [comment]
other than manually testing this---is there a test I can write to include this? Maybe add it to the QS integration test?

**Pull request merged**
